lib/owncloud/test_sharePropagationInside.py

```diff
@@ -0,0 +1,406 @@
+| 6     | propagation     | propagation    | propagation       | propagation | NOT propagation |
++-------+-----------------+----------------+-------------------+-------------+-----------------+
+| 7     |                 |                | upload to         |             |                 |
+|       |                 |                |   -> /test        |             |                 |
++-------+-----------------+----------------+-------------------+-------------+-----------------+
+| 8     | propagation     | propagation    | propagation       | propagation | NOT propagation |
++-------+-----------------+----------------+-------------------+-------------+-----------------+
+| 9     | upload to       |                |                   |             |                 |
+|       |   -> /test/sub  |                |                   |             |                 |
++-------+-----------------+----------------+-------------------+-------------+-----------------+
+| 10    | propagation     | propagation    | propagation       | propagation | propagation     |
++-------+-----------------+----------------+-------------------+-------------+-----------------+
+| 11    |                 | upload to      |                   |             |                 |
+|       |                 |   -> /test/sub |                   |             |                 |
++-------+-----------------+----------------+-------------------+-------------+-----------------+
+| 12    | propagation     | propagation    | propagation       | propagation | propagation     |
++-------+-----------------+----------------+-------------------+-------------+-----------------+
+| 13    |                 |                |                   |             | upload to /sub  |
++-------+-----------------+----------------+-------------------+-------------+-----------------+
+| 14    | propagation     | propagation    | propagation       | propagation | propagation     |
++-------+-----------------+----------------+-------------------+-------------+-----------------+
+| 15    |                 |                | unshare           |             |                 |
+|       |                 |                |   -> /test/sub    |             |                 |
++-------+-----------------+----------------+-------------------+-------------+-----------------+
+| 16    | NOT propagation | NOT            | NOT propagation   | NOT         | propagation     |
+|       |                 | propagation    |                   | propagation |                 |
++-------+-----------------+----------------+-------------------+-------------+-----------------+
+"""
+from smashbox.utilities import *
+import itertools
+import os.path
+import re
+
+@add_worker
+def setup(step):
+
+    step(1, 'create test users')
+    reset_owncloud_account(num_test_users=5)
+    check_users(5)
+
+    reset_rundir()
+    reset_server_log_file()
```

Comment:
```py
    reset_rundir()
    reset_server_log_file()
```
can be removed, smashbox is automatically doing that before each test since end of september